.github/workflows/template-docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Generate template doc
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: generate doc
+        run: |
+          node ./templateDocs.js
```

Comment:
Can we move the script into the `_helpers` directory? It already contains a couple of other scripts